Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -1088,8 +1088,12 @@ private void OnInternalPickingUp(PickingUpItemEventArgs ev)
             if (!ev.IsAllowed)
                 return;
 
-            if (!TrackedSerials.Contains(ev.Pickup.Serial))
-                TrackedSerials.Add(ev.Pickup.Serial);
+            ev.IsAllowed = false;
+
+            TrackedSerials.Remove(ev.Pickup.Serial);
```

Comment:
It already has an tracked serials check (`if (!Check(ev.Pickup) || ev.Player.Items.Count >= 8)`)
And Serial will be added to this HashSet in `Give` method

Reply:
Ok fair points, but why *remove* it from tracked serials, just to re-add it in give? Since this if check is basically useless, we can remove it, but I see no reason to add the TackedSerials.Remove() here.

Reply:
Oh, yes you right, issue was in a `Spawn` method, i'll revert these change it fix it normaly soon